runners/core-java/src/test/java/org/apache/beam/runners/core/SplittableParDoTest.java

```diff
@@ -146,18 +151,23 @@ public void testBoundednessForUnboundedFn() {
         "Applying an unbounded SDF to a bounded collection produces a bounded collection",
         PCollection.IsBounded.UNBOUNDED,
         makeBoundedCollection(pipeline)
-            .apply("unbounded to bounded", new SplittableParDo<>(unboundedFn))
+            .apply("unbounded to bounded", new SplittableParDo.Bound<>(ParDo.of(unboundedFn)))
             .isBounded());
     assertEquals(
         "Applying an unbounded SDF to an unbounded collection produces an unbounded collection",
         PCollection.IsBounded.UNBOUNDED,
         makeUnboundedCollection(pipeline)
-            .apply("unbounded to unbounded", new SplittableParDo<>(unboundedFn))
+            .apply("unbounded to unbounded", new SplittableParDo.Bound<>(ParDo.of(unboundedFn)))
```

Comment:
Note that in this test, I'm testing ProcessFn, rather than a user's splittable DoFn. Agreed that I'll need to make DoFnTester usable with user splittable DoFn's at some point.

Unfortunately I can't have DoFnTester.getOutputWindowedValue, because OutputWindowedValue is declared in runners-core and I'd like to keep it there. It is not a useful part of the SDK; like ProcessFn, it should be part of runners-core.